### PR TITLE
fix(manager_versions.yaml): remove duplicate, leave only base distros

### PR DESCRIPTION
**Pull request by fruch**
we keep updating this file, on every adding of a new distro release it's not really needed, we have only two types of supported RPMs (rhel) and DEBs (debian), and that's enough for picking the repo/list we don't need to map each distro release, since we have it mapped already.

at a point there were diffrent DEB repositories for scylla/manager it was unified years back.

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Review by mikliapko — DISMISSED** [review]

**Review by k0machi — APPROVED** [review]

**Comment by k0machi** [comment]
`test_get_manager_repo_from_defaults_returns_repo_address` probably should be removed or changed.

File: defaults/manager_versions.yaml [hunk 1]
```diff
@@ -1,55 +1,24 @@
...
-    ubuntu22: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
-    ubuntu24: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
+    rhel: 'https://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo'
+    debian: 'https://downloads.scylladb.com/manager/deb/unstable/unified-deb/master/latest/scylla-manager.list'
   "3.3":
-    centos7: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylladb-manager-3.3.repo'
-    centos8: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylladb-manager-3.3.repo'
-    debian10: 'http://downloads.scylladb.com.s3.amazonaws.com/deb/debian/scylladb-manager-3.3.list'
-    debian11: 'http://downloads.scylladb.com.s3.amazonaws.com/deb/debian/scylladb-manager-3.3.list'
-    ubuntu20: 'http://downloads.scylladb.com.s3.amazonaws.com/deb/ubuntu/scylladb-manager-3.3.list'
```
Comment (soyacz) [inline defaults/manager_versions.yaml:15]: these 2 have different links, if they're the same, why releng publish them?

Comment (fruch) [reply defaults/manager_versions.yaml:15]: historical reasons

if you check it's content, it's identical

Comment (fruch) [reply defaults/manager_versions.yaml:15]: once upon a time, scylla wasn't relocatable, and we were maintaining a mirror of the exact same packages we need from each  distro version, and all of our packaging was following the debian structures and naming.

as some point we go unified-deb build, and there's nothing unique about it.

manager was doing the same as core for no actual reasons, and at some point have unified-deb as well.

**Review by soyacz — APPROVED** [review]
LGTM

**Review by k0machi — APPROVED** [review]

**Comment by mikliapko** [comment]
@fruch 
We need to backport it to manager-3.3 branch as well. 
Otherwise, ubuntu24 runs fail.

**Comment by fruch** [comment]
> @fruch 
> We need to backport it to manager-3.3 branch as well. 
> Otherwise, ubuntu24 runs fail.

You don't need to wait for me, mark it for the backport, and push it forward.

**Comment by mikliapko** [comment]
> You don't need to wait for me, mark it for the backport, and push it forward.

Good to know, was not sure that it works the same way for closed PR as well.
Thanks!

**Pull request merged**